source/javascripts/components/unified-editor/StepSelectorDrawer/hooks/useSearch.ts

```diff
@@ -0,0 +1,26 @@
+import { create } from 'zustand';
+
+type State = {
+  searchStep: string;
+  searchStepBundle: string;
+  searchStepCategories: string[];
+};
+
+type Actions = {
+  reset: () => void;
+  setSearchStep: (searchStep: string) => void;
+  setSearchStepBundle: (searchStepBundle: string) => void;
+  setSearchStepCategories: (searchStepCategories: string[]) => void;
+};
+
+const useSearch = create<State & Actions>((set) => ({
+  searchStep: '',
+  searchStepBundle: '',
+  searchStepCategories: [],
```

Comment:
I'd rename these, since they start with a verb, so in the place of usage, I'd assume that these are methods rather than fields.